src/BroadcasttClient.php

```diff
@@ -412,12 +412,12 @@ public function triggerBatch(array $batch = [], bool $jsonEncoded = false): bool
                     throw new JsonEncodeException($event['data'], json_last_error_msg(), json_last_error());
                 }
 
-                $batch[$key]['data'] = $jsonData;
+                $events[$key]['data'] = $jsonData;
             }
         }
 
         $postParams = [];
-        $postParams['batch'] = $batch;
+        $postParams['events'] = $events;
```

Comment:
We should not change the name of the parameter only the method name. However, because of this I found an issue. See my other comment.